solutions/kubernetes/resource/Resource.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash 
```

Comment:
rename file to `setup-metric-server.sh`, folder name `resource` to `resources` and `components.yaml` to `metric-server-components.yaml`
CC - @shailesh-vaidya, please verify or suggest accordingly